### PR TITLE
add initial info about Libreville

### PR DESCRIPTION
**Pull request by koffisani**
- Has someone been in Libreville before ?

### CODE REVIEW

File: koffisani-libreville.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Attractions 
+- Music : Jazze, Ntsame
```
Comment (JohnTheLearner) [inline koffisani-libreville.md:2]: I love jazz music!  :+1:

File: koffisani-libreville.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# Restaurants
+- Pelisson
```
Comment (koffisani) [inline koffisani-libreville.md:5]: Is that place nice ?

File: koffisani-libreville.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+- Parisienne
+
+# Local Activities
+- Trade
```
Comment (JohnTheLearner) [inline koffisani-libreville.md:9]: @koffisani What do you mean by trade?  What kind of trade?

File: koffisani-libreville.md (hunk 2 above)
Comment (koffisani) [inline koffisani-libreville.md:5]: I <3 to go there.

File: koffisani-libreville.md (hunk 3 above)
Comment (koffisani) [inline koffisani-libreville.md:9]: @JohnTheLearner I'm talking about commercial activities.

**Pull request merged**